x-pack/plugins/observability_solution/apm/public/components/app/breadcrumb/index.tsx

```diff
@@ -11,16 +11,22 @@ import { useBreadcrumb } from '../../../context/breadcrumbs/use_breadcrumb';
 export const Breadcrumb = ({
   title,
   href,
+  omitOnServerless = false,
   children,
 }: {
   title: string;
   href: string;
+  omitOnServerless?: boolean;
   children: React.ReactElement;
 }) => {
   const { core } = useApmPluginContext();
+
   useBreadcrumb(
     () => ({ title, href: core.http.basePath.prepend('/app/apm' + href) }),
-    [core.http.basePath, href, title]
+    [core.http.basePath, href, title],
+    {
+      omitOnServerless,
```

Comment:
>[!NOTE]
Side nav items are automatically added, so we have to avoid adding repeated items to the breadcrumbs

<img width="251" alt="image" src="https://github.com/user-attachments/assets/6338a24f-e90a-415f-a90d-e5dd3c5f16dc">